locations/ohlauer.yml

```diff
@@ -1,21 +1,21 @@
 ---
 
 location: ohlauer
-location_nice: Ohlauer Str. 43, 10999 Berlin
+location_nice: Ohlauer
 latitude: 52.49378703855491
 longitude: 13.42969238683751
 altitude: 55
 community: true
 
+local_asn: 44194
```

Comment:
This var should be set for all gaateways

Reply:
I see your point, but whats about the other gateways? If we remove the variable from the group vars we need to add it to every gateway? Or is this the only one where we currently announce this ASN?

Reply:
Ah that's what you mean, yes the other three (ak36, l105, strom) already have their own `local_asn`+`peer_asn`, so with this PR it's consistent